mne/io/brainvision/tests/test_brainvision.py

```diff
@@ -240,8 +239,8 @@ def test_brainvision_data_lowpass_filters():
 
     assert_true(all(any([lp, hp]) for lp, hp in expected_warnings))
 
-    assert_equal(raw.info['highpass'], 0.1)
-    assert_equal(raw.info['lowpass'], 250.)
+    assert_equal(raw.info['highpass'], 1. / (2 * np.pi * 10))
+    assert_equal(raw.info['lowpass'], 1. / (2 * np.pi * 0.004))
```

Comment:
Rather than writing 1/250 as 0.004, I would put the 250 into the numerator to make it clear what the original Hz was. This also make a nice parallel between the Hertz and seconds code.

Reply:
there is no "250" anywhere in the files --> it was computed using the "old formula" to convert seconds to Hertz: `1. / t`. In the files, t=0.004, so that's where the 250 come from. For the code, I suggest to leave it at 0.004 to keep the consistency of the formula `1. / (2 * pi * t)`